scripts/events_module/condition_events.py

```diff
@@ -689,16 +721,28 @@ def handle_already_injured(cat):
 
                 if condition_got is not None:
                     # gather potential event strings for gotten condition
-                    possible_string_list = (
-                        Condition_Events.PERMANENT_CONDITION_GOT_STRINGS[injury][
-                            condition_got
-                        ]
-                    )
 
+                    try:
+                        possible_string_list = (
+                            Condition_Events.PERMANENT_CONDITION_GOT_STRINGS[injury][
+                                condition_got
+                            ]
+                        )
+                    except KeyError:
+                        print(
+                            f"WARNING: No entry in gain_permanent_condition_strings for injury '{injury}' causing "
+                            f"condition '{condition_got}'. Using default."
+                        )
+                        possible_string_list = (
+                            f"After m_c's {injury} healed, {{PRONOUN/m_c/subject}} now {{VERB/m_c/have/has}} "
+                            f"{condition_got}. [Please report this if you see it!]",
```

Comment:
`possible_string_list` needs to be a list, this looks like just a f string, unless I'm misunderstanding something?

Reply:
Ughhhh the IDE changed my bracketssss